### PR TITLE
Feat/add app changelog for user

### PR DESCRIPTION
**Pull request by tomasz-trela**

### CODE REVIEW

File: lib/features/app_changelog/show_changelog.dart [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+import "package:package_info_plus/package_info_plus.dart";
+
+import "app_changelog.dart";
+import "repository/changelog_repository.dart";
+import "repository/local_changelog_repository.dart";
+
+Future<void> showChangelog(BuildContext context, WidgetRef ref) async {
+  final changelogs = await ref.watch(changelogRepositoryProvider.future);
```
Comment (simon-the-shark) [inline lib/features/app_changelog/show_changelog.dart:11]: not sure if calling `ref.watch` outside `build` method of either a `Widget` or other riverpod's provider is safe. `ref.read` should be more appropiate in this case.

File: lib/features/app_changelog/widgets/app_changelog_list.dart [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  final Changelog changelog;
+
+  @override
+  Widget build(BuildContext context) {
+    return Flexible(
+      child: ListView.separated(
+        shrinkWrap: true,
+        itemBuilder: (BuildContext context, int index) => _ListItem(
+          tag: changelog.changes![index]!.changeType,
```
Comment (simon-the-shark) [inline lib/features/app_changelog/widgets/app_changelog_list.dart:17]: uhh - I lot of `!` signs. I do not like it - we should avoid force unwrapping in all cases possible, just to be safe

Comment (simon-the-shark) [reply lib/features/app_changelog/widgets/app_changelog_list.dart:17]: you can think of some default values for some fields or hide the whole ui list item if there's a null in some important place. do as you please, but something more safe please

File: lib/features/offline_messages/messages_config.dart [hunk 3]
```diff
@@ -16,6 +16,7 @@ extension GqlOfflineMessageX on BuildContext {
           localize.offline_department_details,
         TtlKey.guideDetailsRepository => localize.offline_guide_details,
         TtlKey.guideRepository => localize.offline_guide_posts,
+        TtlKey.changelogRepository => "",
```
Comment (simon-the-shark) [inline lib/features/offline_messages/messages_config.dart:19]: let's give it some proper label :(

File: lib/features/home_view/home_view.dart [hunk 4]
```diff
@@ -27,17 +29,20 @@ class HomeView extends StatelessWidget {
...
-    return Scaffold(
-      backgroundColor: context.colorTheme.whiteSoap,
-      appBar: LogoAppBar(context),
-      body: KeepAliveHomeViewProviders(
-        child: ListView.separated(
-          padding: const EdgeInsets.only(bottom: HomeViewConfig.bottomPadding),
-          itemBuilder: (context, index) => sections[index],
-          separatorBuilder: (context, index) => SizedBox(
-            height: index == 1 ? 0 : HomeViewConfig.paddingMedium,
+    return UpdateChangelogWrapper(
```
Comment (simon-the-shark) [inline lib/features/home_view/home_view.dart:32]: have you tried placing this somewhere in the rootview instead? did it work?

File: lib/features/app_changelog/repository/get_newest_changelog.graphql [hunk 5]
```diff
@@ -0,0 +1,16 @@
+query GetChangelogs(
```
Comment (simon-the-shark) [inline lib/features/app_changelog/repository/get_newest_changelog.graphql:1]: why do we have the same query twice in the filesystem?

File: lib/features/app_changelog/repository/get_changelog.graphql [hunk 6]
```diff
@@ -0,0 +1,16 @@
+query GetChangelog(
```
Comment (simon-the-shark) [inline lib/features/app_changelog/repository/get_changelog.graphql:1]: why do we have the same query twice in the filesystem?

**Review by simon-the-shark — COMMENTED** [review]
I think you've done really nicely with this task :) I've left few notes and questions, but they're are just a few small details.

**Comment by tomasz-trela** [comment]
Strange things happened during the rebase

**Comment by simon-the-shark** [comment]
> Strange things happened during the rebase

yeah, basically something went wrong, but no worry, we'll squash anything at the end anyways

**Comment by tomasz-trela** [comment]
Now it should be good :)

**Pull request merged**